engine/src/main/java/org/camunda/bpm/engine/impl/telemetry/dto/InternalsImpl.java

```diff
@@ -39,10 +40,10 @@ public class InternalsImpl implements Internals {
   protected ApplicationServerImpl applicationServer;
   @SerializedName(value = SERIALIZED_LICENSE_KEY)
   protected LicenseKeyDataImpl licenseKey;
-  protected Map<String, Command> commands;
   @SerializedName(value = SERIALIZED_CAMUNDA_INTEGRATION)
   protected Set<String> camundaIntegration;
-
+  protected Date dataCollectionStartDate;
```

Comment:
I think adding `@SerializedName` is required so the field name is consistent with the rest of the fields.
If this change is applied, the REST API, docs, and Open API have to be updated as well.

Reply:
I also adjusted it in the REST dto. This is where we actually specify the name for the JSON that is returned by the REST endpoint.